client/src/components/AddMovieForm/AddMovieForm.js

```diff
@@ -0,0 +1,112 @@
+import { useState } from 'react'
+import styled from 'styled-components/macro'
+import axios from 'axios'
+import ButtonGreen from '../Buttons/ButtonGreen/ButtonGreen'
+import SuccessCard from '../Messages/SuccessCard/SuccessCard'
+import ErrorCard from '../Messages/ErrorCard/ErrorCard'
+
+const AddMovieForm = () => {
+  const [tmdbId, setTmdbId] = useState('')
+  const [successMessage, setSuccessMessage] = useState('')
+  const [errorMessage, setErrorMessage] = useState('')
+
+  // Comments will be deleted later on
+  /*
+    The function has to be async, because it has to wait for the response of the tmdb api in the try catch block.
+    Once it has an response, the response can be positive or negative.
+    If the response is positive we allow (allowPostMovie) the movie to be posted to our database.
+    If the response is negative we set an error message.
+  */
+  const addMovieHandler = async event => {
+    event.preventDefault()
+    event.target.reset()
```

Comment:
Just food for thought: do you want to reset the form even if the submit event does not work? It may be nicer for the user to see their input if an error occurred. Does that make sense?

Reply:
Yes makes sense, changed!